Changes

```diff
@@ -1,5 +1,8 @@
 LIST OF CHANGES FOR NPG-QC PACKAGE
 
+ - SeqQC
+   - bower configuration in npg_qc_viewer root
```

Comment:
As discussed, 'root' is ambiguous due to the presence of 'root' directory. Please rephrase.